website/docs/r/ess_scaling_rule.html.markdown

```diff
@@ -50,10 +106,18 @@ The following arguments are supported:
 
 The following attributes are exported:
 
-* `id` - The combined scaling rule ID, [scalingGroupId:scalingRuleId].
+* `id` - The combined scaling rule ID.
 * `scaling_group_id` - The id of scaling group.
 * `ari` - Unique identifier of a scaling rule.
 * `adjustment_type` - Adjustment mode of a scaling rule.
 * `adjustment_value` - Adjustment value of a scaling rule.
 * `scaling_rule_name` - Name of a scaling rule.
-* `cooldown` - Cool-down time of a scaling rule.
\ No newline at end of file
+* `cooldown` - Cool-down time of a scaling rule.
```

Comment:
If one parameter is a input, there is no need to define it again in exported block. Please remove line 110-115.